src/utils/data/animals.mock.json

```diff
@@ -1,250 +1,232 @@
       "type": "data"
     },
     {
-      "id": "2cd4ba93",
-      "name": "pig",
-      "full_name": "pig",
-      "tags": ["medium"],
-      "layer": "Feature",
-      "pipelines": ["__default__", "ds"],
+      "full_name": "horse",
+      "id": "091b5035",
+      "layer": "model output",
+      "name": "Horse",
+      "pipelines": ["de", "__default__"],
+      "tags": ["small"],
       "type": "data"
-    }
-  ],
-  "edges": [
-    {
-      "target": "4ffcf321",
-      "source": "9d989e8d"
-    },
-    {
-      "target": "4ffcf321",
-      "source": "e4951252"
     },
     {
-      "target": "4ffcf321",
-      "source": "f1f1425b"
+      "full_name": "parameters",
+      "id": "f1f1425b",
+      "layer": null,
+      "name": "Parameters",
+      "pipelines": ["de", "__default__"],
+      "tags": ["small"],
+      "type": "parameters"
     },
     {
-      "target": "4ffcf321",
-      "source": "2a31900d"
+      "full_name": "params:rabbit",
+      "id": "c38d4c6a",
+      "layer": null,
+      "name": "Params:rabbit",
+      "pipelines": ["de", "__default__"],
+      "tags": ["small"],
+      "type": "parameters"
     },
     {
-      "target": "2cd4ba93",
-      "source": "4ffcf321"
+      "full_name": "pig",
+      "id": "2cd4ba93",
+      "layer": "feature",
+      "name": "Pig",
+      "pipelines": ["de", "ds", "__default__"],
+      "tags": ["large", "medium"],
+      "type": "data"
     },
     {
-      "target": "091b5035",
-      "source": "4ffcf321"
+      "full_name": "sheep",
+      "id": "6525f2e6",
+      "layer": "primary",
+      "name": "Sheep",
+      "pipelines": ["de", "ds", "__default__"],
+      "tags": ["small"],
+      "type": "data"
     },
     {
-      "target": "6525f2e6",
-      "source": "4ffcf321"
+      "full_name": "weasel",
+      "id": "85c4cf64",
+      "layer": "raw",
+      "name": "Weasel",
+      "pipelines": ["de", "__default__"],
+      "tags": ["large", "medium"],
+      "type": "data"
     },
     {
-      "target": "4f90af66",
-      "source": "9d989e8d"
+      "full_name": "trout",
+      "id": "e27376a9",
+      "name": "trout",
+      "pipelines": ["ds", "__default__"],
+      "tags": [],
+      "type": "task"
     },
     {
-      "target": "4f90af66",
-      "source": "85c4cf64"
-    },
+      "full_name": "whale",
+      "id": "1769e230",
+      "layer": "model output",
+      "name": "Whale",
+      "pipelines": ["ds", "__default__"],
+      "tags": [],
+      "type": "data"
+    }
+  ],
+  "pipelines": [
     {
-      "target": "4f90af66",
-      "source": "0ae9e4de"
+      "id": "de",
+      "name": "De"
     },
     {
-      "target": "4f90af66",
-      "source": "09f5edeb"
+      "id": "ds",
+      "name": "Ds"
     },
     {
-      "target": "6525f2e6",
-      "source": "4f90af66"
+      "id": "__default__",
+      "name": "Default"
     },
     {
-      "target": "2cd4ba93",
-      "source": "4f90af66"
-    },
+      "id": "empty",
+      "name": "Empty"
+    }
+  ],
```

Comment:
@921kiyo Is there any way that we can keep the full pipeline display names here? e.g. `De` -> `Data engineering`, as it was?

Reply:
@richardwestenra Unfortunately, it doesn't happen at the moment. `name` value is a prettified version of `id`. So `id` would have to be `data engineering` if we wanted `Data engineering` for `name`

Reply:
gotcha. ok no biggie! thanks for the quick explanation :)